src/analysis/utility.jl

```diff
@@ -0,0 +1,19 @@
+"""
+Convert an array of tuples to a 2D array. Presumes that the tuples are all the same
+length and type (there is currently no check).
+"""
+# can't figure out how to pre-set the type!
+#function array_from_tuples(T::Array{Tuple{Vararg{Number},1})
+function array_from_tuples(T) 
```

Comment:
Would `T::Array{Tuple}` work?

Reply:
It doesn't:
```
ERROR: LoadError: MethodError: no method matching array_from_tuples(::Array{Tuple{Float64,Float64},1})
Closest candidates are:
  array_from_tuples(::Array{Tuple,N} where N) at /Users/jstickel/.julia/dev/WaterModelsAnalytics/src/analysis/utility.jl:7
Stacktrace:
```
I'm stumped on this one.

Reply:
Oh, maybe try this: `T::Array{Tuple{<:Any, <:Any}}`

Reply:
That doesn't work either. `T::Array{Tuple{Float64, Float64}, 1}` works, but `T::Array{Tuple{Number, Number}, 1}` or `T::Array{Tuple{<:Number, <:Number}, 1}` does not. Also, `T::Array{Tuple{Vararg{Float64}},1}` does not work either, which would allow tuples longer than 2 to be provided. I'm ready to let this one go...

Reply:
FYI, this did it: `arrtup::Array{Tuple{Vararg{T,N}},1} where {T,N}` (also changed the variable name, but that wasn't the issue.)